scripts/load-logs.js

```diff
@@ -30,11 +35,12 @@ function createBackgroundWindow() {
       nodeIntegration: true
     }
   });
-  win.loadURL(`file://${__dirname}/../window_background/index.html`);
+  win.loadURL(`file://${__dirname}/../lib/window_background/index.html`);
```

Comment:
this sould be under `/src`;
```suggestion
  win.loadURL(`file://${__dirname}/../src/window_background/index.html`);
```

Reply:
arguably you want the transpiled version? though it's not obvious that it's a dependency...

Reply:
So pre-running the transpiler is a requirement to run scripts, should we force it before running the script?

Reply:
I tried `/src/` at first and it didn't work. If that change is made without anything else then it will break. I am not familiar enough with babel to know the correct other changes to make.